### PR TITLE
[Brent] Initial setup

### PR DESCRIPTION
**Pull request by dracos**
Think this is the skeleton of what's needed, barring additional information to come in about e.g. XML updates, special cases to implement.

### CODE REVIEW

**Comment by chrismytton** [comment]
Just to note that I've done some very similar changes for Camden in https://github.com/mysociety/open311-adapter/pull/233, so there may be some overlap/conflicts when they both get merged into master.

File: conf/council-brent_symology.yml-example [hunk 1]
```diff
@@ -1,4 +1,6 @@
 endpoint_url: ''
+endpoint_username: ''
+endpoint_password: ''
```
Comment (chrismytton) [inline conf/council-brent_symology.yml-example:3]: Is the endpoint username different from the `username: FMS` that we have for Bexley/Central Beds (that's used in `perllib/Open311/Endpoint/Integration/Symology.pm`, or are they the same? Just a bit wary of having two usernames floating around!

**Review by chrismytton — APPROVED** [review]
One question about possible confusion around multiple Symology usernames, but otherwise this looks good to go!

File: conf/council-brent_symology.yml-example (hunk 1 above)
Comment (dracos) [reply conf/council-brent_symology.yml-example:3]: I'm afraid they are different, yes. One is the username for connecting/access to the endpoint, the other is the in-Symology username of the user making the request.

**Pull request merged**